src/runtime-rs/crates/service/src/manager.rs

```diff
@@ -159,39 +159,20 @@ impl ServiceManager {
         Ok(())
     }
 
-    async fn send_event(&self, event: Arc<dyn Event>) -> Result<()> {
-        let any = Any {
-            type_url: event.type_url(),
-            value: event.value().context("get event value")?,
-            ..Default::default()
+    async fn new_event_publisher(namespace: &str) -> Result<Box<dyn Forwarder>> {
```

Comment:
I don't think it's better to move the `new_event_publisher`  to `event.rs` and the `manager.rs` doesn't need to be aware of `ContainerdForwarder` or `LogForwarder`.

Reply:
Thanks for your suggestion. It has been moved to `event.rs`. PTAL.